charts/flagger/values.yaml

```diff
@@ -23,6 +23,15 @@ podPriorityClassName: ""
 
 metricsServer: "http://prometheus:9090"
 
+# creates service for 'http' port
+service:
+  enabled: false
+  port: 8080
```

Comment:
Let's delete this and create the service only when serviceMonitor is enabled

Reply:
@stefanprodan done, please review

Reply:
Please add serviceMonitor to the chart readme, then squash all your commits into a single one.

Reply:
@stefanprodan done